### PR TITLE
Update zh_cn

### PR DESCRIPTION
**Pull request by ChuijkYahus**

### CODE REVIEW

File: src/main/resources/assets/hexbound/lang/zh_cn.flatten.json5 [hunk 1]
```diff
@@ -177,26 +177,26 @@
...
         },
         casting: {
             figments: {
                 title: "虚意识",
                 "1": "我之前对相关学说的所有研究都表明，$(hex)咒术$()的施放必须要有一个存在于$(o)世界$()上的、实在的施法者。\
-|                     显然，之前所有的$(hex)咒术$()都是由我施放的，满足上述要求。但就算\
-|                     最终找到了$(l:hexcasting:greatwork/spellcircles)某些独立于施法者的施法方式$()（或者改装一下发射器让它能甩动缀品说不定也行），\
-|                     它们也会在我跨过前往其他维度的传送门时断开连接并停止工作。",
+                      显然，之前所有的$(hex)咒术$()都是由我施放的，满足上述要求。但就算\
+                      最终找到了$(l:hexcasting:greatwork/spellcircle)某些独立于施法者的施法方式$()（或者改装一下发射器让它能甩动缀品说不定也行），\
```
Comment (object-Object) [inline src/main/resources/assets/hexbound/lang/zh_cn.flatten.json5]: I think this is supposed to be `hexcasting:greatwork/spellcircles`.

File: src/main/resources/assets/hexbound/lang/zh_cn.flatten.json5 [hunk 2]
```diff
@@ -234,25 +234,25 @@
         },
         "greatwork.impetus_figment": {
             title: "促动石用虚意识",
-            "1": "如今我已有控制意识的能力，我又读了一遍有关$(l:hexbound:patterns/great_spells/impetus_fake_player)绑定虚意识$()的笔记，并确认了$(l:hexcasting:greatwork/impetus)牧师促动石$()是虚意识的极佳容器。\
-|                 该促动石需绑定至一个和我一样已受$(o)启迪$()的$(hex)咒术师$()。也没有其他意识能将自身的一部分抽离到这种程度了。$(br2)目前还没有。但他们会的。迟早会的。",
+            "1": "如今我已有控制意识的能力，我又读了一遍有关$(l:hexbound:patterns/great_spells/impetus_fake_player)绑定虚意识$()的笔记，并确认了$(l:hexcasting:greatwork/impetuses)牧师促动石$()是虚意识的极佳容器。\
```
Comment (object-Object) [inline src/main/resources/assets/hexbound/lang/zh_cn.flatten.json5]: I think this is supposed to be `hexcasting:greatwork/impetus`.

**Review by object-Object — CHANGES_REQUESTED** [review]
Thanks for the update! I'm sorry for the delay, I didn't see this at the time because I forgot to enable notifications for this repo. It seems like a couple of the book links are broken.

**Comment by ChuijkYahus** [comment]
Sorry for the broken links, fixed now.

**Review by object-Object — APPROVED** [review]
thanks!

**Pull request merged**